packages/modeling/src/operations/booleans/scission.js

```diff
@@ -27,7 +27,6 @@ import { scissionGeom3 } from './scissionGeom3.js'
  */
 export const scission = (...objects) => {
   objects = flatten(objects)
-  if (objects.length === 0) throw new Error('wrong number of arguments')
```

Comment:
Seems a bit strange to remove this throw when throw are being added to other functions.

Reply:
I reverted this change. We can discuss removing in the next PR :-)